src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,9 @@
+import styles from './Navbar.module.scss';
+
+export default function Navbar(): JSX.Element {
+  return (
+    <div className={styles['Navbar__container']}>
+      <button className={styles['Navbar__button']}>Do Something</button>
+    </div>
+  );
+}
```

Comment:
My comments about this apply to every single instance.

This is very much a stylistic issue and I would prefer to have named exports rather than default exports. 
Reasons for that are: 
- export * from "./x" is immediately handled with naming and so you can do multiple imports in import {...} from './x', not a big deal with components that only export 1 thing but still just looks better imo
- doing this means you have HAVE to name your default exports, I won't budge on that which is just more work than export *

Reply:
Didn't realize it would only grab the bottom portion, im used to bitbucket, but talking about first line to export and create the function not the actual code shown here. Same with the bottom comment

Reply:
Please see https://github.com/UNLV-CS472-672/2024-S-GROUP1-Roadwatch/pull/12#discussion_r1509897794